signal-exit dependency issue on version 9.1.0

<!-- Please answer the following questions. Issues that don't use this template will be closed. -->

> Possible bug with signal-exit dependency

When installing the last version via NPM and running the commands via CLI signal-exit module seems to be missing.

> Which rule, if any, is this issue related to?

No rule

> What CSS is needed to reproduce this issue?

None CSS involved yet.

> What stylelint configuration is needed to reproduce this issue?

No rule involved yet

> Which version of stylelint are you using?

9.1.0

> How are you running stylelint: CLI, PostCSS plugin, Node API?

e.g. "CLI with `stylelint "src/*.scss" --syntax scss" 

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

No

> What did you expect to happen?

No errors of missing dependencies should appear on console

> What actually happened (e.g. what warnings or errors you are getting)?

```
stylelint "src/*.scss" --syntax scss
module.js:327
    throw err;
    ^
Error: Cannot find module 'signal-exit'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (C:\Users\dgoncalves\AppData\Roaming\npm\node_modules\stylelint\lib\vendor\writeFileAtomic.js:13:14)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)`

```

So downgrading to version 8.4.0 works, the require on writeFileAtomic.js is unable to found 'signal-exit' I tried installing it manually and including it in my package.json but it doesn't work that way either. This seems to happen on the last version.

I tried clearing cache, removing global and local node_modules but none of them worked

--- a/package.json
+++ b/package.json
@@ -71,6 +71,7 @@
     "postcss-selector-parser": "^3.1.0",
     "postcss-value-parser": "^3.3.0",
     "resolve-from": "^4.0.0",
+    "signal-exit": "^3.0.2",
     "specificity": "^0.3.1",
     "string-width": "^2.1.0",
     "style-search": "^0.1.0",
@@ -102,7 +103,6 @@
     "remark-preset-lint-recommended": "^3.0.0",
     "remark-validate-links": "^7.0.0",
     "request": "^2.81.0",
-    "signal-exit": "^3.0.2",
     "strip-ansi": "^4.0.0",
     "weak": "^1.0.1"
   },

